crates/rpc-types-beacon/src/relay.rs

```diff
@@ -158,9 +159,9 @@ pub struct SignedBidSubmissionV4 {
     /// The Electra block bundle for this bid.
     pub blobs_bundle: BlobsBundleV1,
     /// The Pectra execution requests for this bid.
-    pub execution_requests: Vec<Bytes>,
+    pub execution_requests: ExecutionRequestsV4,
     /// The EIP-7742 blobs per block for this bid.
-    #[serde(with = "alloy_serde::quantity")]
+    #[serde_as(as = "DisplayFromStr")]
     pub target_blobs_per_block: u64,
```

Comment:
Aside: I saw today that EIP-7742 was pulled from Pectra, my PR might conflict with removing that since I changed `SignedBidSubmissionV4::target_blobs_per_block` to `DisplayFromStr` as well.

https://ethereum-magicians.org/t/all-core-devs-consensus-acdc-147-december-12-2024/22161/2